docs/en/pid-eaa-issuance.rst

```diff
@@ -517,6 +517,9 @@ The ``request`` JWT payload contained in the HTTP POST message is given with the
     * - **code_challenge_method**
       - A method that was used to derive **code challenge**. It MUST be set to ``S256``.
       - :rfc:`7636#section-4.3`.
+    * - **scope**
+      - JSON String. String specifying a unique identifier of the Credential being described in the `credential_configurations_supported` map in the Credential Issuer Metadata. For example, in the case of the PID, it MUST be set to ``PersonIdentificationData``. It MAY be multivalued, each value must be separated by a space.
```

Comment:
```suggestion
      - JSON String. String specifying a unique identifier of the Credential being described in the `credential_configurations_supported` map in the Credential Issuer Metadata. For example, in the case of the PID, it MUST be set to ``PersonIdentificationData``. It MAY be multivalued, each value MUST be separated by a space.
```